### PR TITLE
ibus-libpinyin: init at 1.7.4

### PR DESCRIPTION
**Pull request by yokto**
###### Things done:
- [ ] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/ibus-engines/ibus-libpinyin/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  #configureFlags = "--with-anthy-zipcode=${anthy}/share/anthy/zipcode.t";
+
+  buildInputs = [
+  ibus glib sqlite libpinyin python3 gtk3 db
+  ];
+
+  nativeBuildInputs = [ intltool pkgconfig ];
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/libpinyin/ibus-libpinyin/ibus-libpinyin-${version}.tar.gz";
```
Comment (jagajaga) [inline pkgs/tools/inputmethods/ibus-engines/ibus-libpinyin/default.nix]: Please use `mirror://`.

File: pkgs/tools/inputmethods/libpinyin/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    description = "The libpinyin project aims to provide the algorithms core for intelligent sentence-based Chinese pinyin input methods.";
+    homepace    = https://sourceforge.net/projects/libpinyin;
+    license     = licenses.gpl2;
+    platforms   = platforms.linux;
+  };
+
+  buildInputs = [ glib db pkgconfig ];
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/libpinyin/libpinyin/libpinyin-1.3.0.tar.gz";
```
Comment (jagajaga) [inline pkgs/tools/inputmethods/libpinyin/default.nix]: Same here.

File: pkgs/tools/inputmethods/libpinyin/default.nix [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, glib, db, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "libpinyin-1.3.0";
+
+  meta = with stdenv.lib; {
+    description = "The libpinyin project aims to provide the algorithms core for intelligent sentence-based Chinese pinyin input methods.";
```
Comment (fpletz) [inline pkgs/tools/inputmethods/libpinyin/default.nix]: This should be a rather short description. Maybe "Library for intelligent sentence-based Chinese pinyin input method"?

**Comment by yokto** [comment]
Will that be all? I would prefer to do all the necessary changes at once, if possible. Thank you.

**Comment by joachifm** [comment]
Please change the description per @fpletz's suggestion, otherwise this looks ok.

**Comment by yokto** [comment]
Should I squash all commits belonging to the same package and push over the branch. Or do you prefer to have all the commits as they were discussed here.

**Comment by joachifm** [comment]
Please squash the fixups into their respective commits. Fixups are usually not that helpful to a person reading the history later on.

**Pull request merged**